src/components/features/CountComments/CountComments.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+class CountComments extends React.Component {
+  render() {
+    const { posts } = this.props;
+    let numberOfComments = [];
+    posts.forEach(element => {
+      numberOfComments.push(element.comments.length);
+    });
```

Comment:
To jest błędny sposób liczenai sumy, ale tutaj nie będzie to potrzebne, komponenet otrzyma tylko tablicę komentarzy z której łatwo pobrać długość